packages/admin/blocks-admin/src/blocks/types.tsx

```diff
@@ -76,6 +76,8 @@ export interface BlockMethods<
     previewContent: (state: State, context?: BlockContext) => PreviewContent[];
     dynamicDisplayName?: (state: State) => React.ReactNode;
     anchors?: (state: State) => string[];
+    extractTextContents?: (state: State) => string[];
+    replaceTextContents?: (state: State, contents: Array<{ original: string; replaceWith: string }>) => OutputApi;
```

Comment:
Shouldn't replaceTextContents return a new state instead of this output? Using the output makes the method inflexible